actions/class.Runner.php

```diff
@@ -168,10 +181,11 @@ protected function getServiceContext()
             }
             $testDefinition = $container->getSourceTest($execution);
             $testCompilation = $container->getPrivateDirId($execution) . '|' . $container->getPublicDirId($execution);
+
             $this->serviceContext = $this->getRunnerService()->getServiceContext(
                 $testDefinition,
                 $testCompilation,
-                $testExecution
+                $execution->getIdentifier()
```

Comment:
Before it was the `sessionId`, now it is the `deliveryExecution URI`. Wont it break compatibility with other places?

Reply:
The docblock for `getServiceContext()` states it expects _The URI of the delivery execution_ for this parameter, so I think this is correct. However, I don't remember anymore the reason for this change (maybe there was something not working), I'll recheck.

Reply:
Indeed it was and as you can see [here](https://github.com/oat-sa/extension-tao-testqti/pull/2421/files#diff-fea95e24360d827ee2d0e75d5c4d18bc5d43134b522acb2f205ad18f7be19a67L152) it uses the sessionId to retrieve the execution, so it seems they are the same thing. 

Although, if this change is not required, I would not touch it to avoid possible issues.

Reply:
Indeed, both take the same value:

```
[2023-11-27 15:50:06] tao.CRITICAL: testExecution: http://backoffice.docker.localhost/ontologies/tao.rdf#i6564ac934f79397b0c319ed0b1335e execution.identifier: http://backoffice.docker.localhost/ontologies/tao.rdf#i6564ac934f79397b0c319ed0b1335e [] []
```

I'll rollback to get the value as we did previously.

Reply:
Rolled back

Reply:
Thanks!